src/main/java/org/broadinstitute/hellbender/utils/IntervalUtils.java

```diff
@@ -147,22 +167,23 @@ public static int compareContigs(final Locatable first, final Locatable second,
 
     /**
      * getSpanningInterval returns interval that covers all of the locations passed in.
+     *
      * @param locations the locations to be spanned (on a single contig)
      * @return the minimal span that covers all locations (could be null if no locations are passed in).
      * @throws IllegalArgumentException if the argument is null
-     * or if the argument contains any null element
-     * or if the locations are not all on the same contig (compared by String.equals)
+     *                                  or if the argument contains any null element
+     *                                  or if the locations are not all on the same contig (compared by String.equals)
```

Comment:
Your IDE appears to have reformated this entire class -- this is bad because we want to preserve accurate provenance information for our code. Can you fix?